x-pack/plugins/observability_solution/observability/public/pages/alert_details/components/related_alerts.tsx

```diff
@@ -79,6 +101,8 @@ export function InternalRelatedAlerts({ alert, kuery }: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [alertStart, alertEnd]);
 
+  if (!kuery || !alert) return <EmptyState />;
```

Comment:
This should never happen because we have filter based on rule id in the kuery but I just added it in case there was a bug.